README.md

```diff
@@ -11,3 +11,13 @@ The deployment is automatic on push to the `master` branch.
 - `npm start`
 
 The frontend will be running on http://localhost:3000/.
+
+## Run tests
```

Comment:
Can you create a testing section containing instructions on running **Unit tests**, **E2E tests** and **Component tests**?